t/discovery/consul_dump.t

```diff
@@ -75,9 +75,11 @@ discovery:
   consul:
     servers:
       - "http://127.0.0.1:8500"
+    skip_services:
+      - "consul"
     dump:
       path: "consul.dump"
-      load_on_init: true
+      load_on_init: false
```

Comment:
Why change this value?

Reply:
for the first time to start apisix, it will print some error log. Just for better reading log